2200062/문제2.py

```diff
@@ -0,0 +1,24 @@
+# 파리 퇴치
+# M크기의 파리채로 N 행렬을 순회..?
+
+T = int(input())
+dx = [0, 1, 1]   # 우, 좌하, 우하
+dy = [1, 0, 1]
+
+for _ in range(1, T+1):
+    N, M = map(int, input(). split())
+    matrix = [list(map(int, input(). split()))for _ in range(N)]
+    death = 0
+    fly = []
+
+    for x in range(N):
+        for y in range(N):
+
+            for i in range(M):
+                nx = x + dx[i]
+                ny = y + dy[i]
+                death += matrix[nx][ny]   
+
+                if 0 <= nx < N and 0 <= ny < N:
+                    x = nx
+                    y = ny    
```

Comment:
x, y 시작점 설정까지 잘해주셨네요! 이제 m * m 칸을 순회하면서 답을 구해주셔야 하는데요,
```suggestion
            death = 0
            for i in range(M):
                for j in range(M):
                    nx = x + i
                    ny = y + j
                    if 0 <= nx < N and 0 <= ny < N:
                        death += matrix[nx][ny]
            fly.append(death)
```
이런 식으로 작성해주시면 m * m 칸을 순회하면서 파리의 수를 구할 수 있습니다 😄